glibc/riscv64.patch

```diff
@@ -1,31 +1,47 @@
---- PKGBUILD
-+++ PKGBUILD
-@@ -7,13 +7,13 @@
+Index: PKGBUILD
+===================================================================
+--- PKGBUILD    (revision 448151)
++++ PKGBUILD    (working copy)
+@@ -7,7 +7,7 @@
  # NOTE: valgrind requires rebuilt with each major glibc version
  
  pkgbase=glibc
 -pkgname=(glibc lib32-glibc)
 +pkgname=(glibc)
  pkgver=2.35
- pkgrel=2
+ _commit=d7d1eebd4d5806be55ffacbf18917ad68d4ae7fd
+ pkgrel=6
+@@ -14,13 +14,11 @@
  arch=(x86_64)
  url='https://www.gnu.org/software/libc'
  license=(GPL LGPL)
 -makedepends=(git gd lib32-gcc-libs python)
 +makedepends=(git gd python)
- optdepends=('perl: for mtrace')
- options=(!strip staticlibs !lto)
- #_commit=3de512be7ea6053255afed6154db9ee31d4e557a
-@@ -36,7 +36,7 @@ b2sums=('623c728884f070cd87ffeb9203f74206197c52405ac9bc44f3dd519a3468b8e8ae2536c
+ options=(debug staticlibs !lto)
+ source=(git+https://sourceware.org/git/glibc.git#commit=${_commit}
+         locale.gen.txt
+         locale-gen
+-        lib32-glibc.conf
+-        sdt.h sdt-config.h
```

Comment:
Better to not remove these (and checksums below). Just leave them as is, though we don't use them, making fewer context changes needed in the future.

Reply:
OK, I will deal with this problem after issues I mentioned above being solved.